docker/Dockerfile.debian

```diff
@@ -0,0 +1,32 @@
+FROM golang AS builder
+
+RUN mkdir /src
+ADD . /src
+
+RUN cd /src && \
+	mkdir -p output/bin && \
+	make V=1 bin/step
+
+FROM debian:bullseye
+
+ENV STEP="/home/step"
+ENV STEPPATH="/home/step"
+ARG STEPUID=1000
+ARG STEPGID=1000
+
+RUN apt-get update \
+        && apt-get upgrade \
+        && apt-get install -y --no-install-recommends bash curl tzdata \
```

Comment:
`bash` and `tzdate` are in the default `debian:bullseye` image.

I guess curl is a nice-to-have thing.